### ISSUE TITLE
Request to bring Delta Chat Desktop into compliance with web content accessibility standards

### ISSUE THREAD

**Issue by bmivzkrp** [deltachat/deltachat-desktop#2845]
Dear developers!
First of all - thank you very much for Delta Chat.
Could you improve the accessibility of Delta Chat Desktop so that blind users can comfortably use this great application? For Linux users, Delta Chat is also another Thunderbird alternative (there are very few email clients for Linux that support accessibility).
Delta Chat Desktop currently has issues that make it difficult for blind users to use. For example - some elements do not have text descriptions (labels), some elements cannot be activated using the Enter key, and so on.
so I have a request for you: could you bring Delta Chat Desktop into line with web accessibility standards?
It is probably better to take the WCAG 2.1 standard as a basis, since WCAG 2.2 and WCAG 3.0 have not yet been approved.
WCAG 2.1 link:
https://www.w3.org/TR/WCAG21/
WCAG 2.1 video:
https://www.youtube.com/watch?v=wmSM-Q0I6Dc
Electronjs Accessibility Documentation:
https://www.electronjs.org/docs/latest/tutorial/accessibility
Many thanks for your attention to this issue.

**Comment by Jikstra** [comment]
We would love to see deltachat being compliant with this but sadly we currently don't have the resources and expertise to tackle this. Our current focus is on getting our backend working in rust so that deltachat can work in normal browsers too and isn't limited to electron. Sadly the electron accessibility tools are not that great from what I know and we could benefit from having all the tools other browsers like firefox support. Another problem currently is that our keybinding support (what would be needed for eg enter press) is very hacky and it can't distinguish between which dialogs are shown and on which level. So it's on our list of things to do but we have some more ground work to do. In case anybody wants to work on this right now we would love to give support and prs are of course very appreciated! It's not easy for us to say this, as it's something that hurts us. We would love to see accessibility as a fundamental part of deltachat, but currently we can't offer it but hope as soon as possible.

**Comment by bmivzkrp** [comment]
@Jikstra
Thank you for your answer.
I would also like to see Delta Chat in the form of a web interface. As a 
blind Linux user, I can say that sometimes certain accessibility 
regressions appear in Chromium, and then they move to Electron, and you 
have to wait for the developers to fix it, and then also for the moment 
when the developers of a certain application move it to a newer version 
Electron version.
Anyway - if you bring future versions of Delta Chat to accessibility 
standards, I can help you in the context of user experience. You can ask 
me some things. If it s a technical point that I don t understand, I can 
ask the developers I know for help.

03.08.22 00:54, Jikstra пише:
> We would love to see deltachat being compliant with this but sadly we currently don't have the resources and expertise to tackle this. Our current focus is on getting our backend working in rust so that deltachat can work in normal browsers too and isn't limited to electron. Sadly the electron accessibility tools are not that great from what I know and we could benefit from having all the tools other browsers like firefox support. Another problem currently is that our keybinding support (what would be needed for eg enter press) is very hacky and it can't distinguish between which dialogs are shown and on which level. So it's on our list of things to do but we have some more ground work to do. In case anybody wants to work on this right now we would love to give support and prs are of course very appreciated! It's not easy for us to say this, as it's something that hurts us. We would love to see accessibility as a fundamental part of deltachat, but currently we can't offer it but hope as soon as possible.
>

**Comment by Simon-Laux** [comment]
This issue is already very old, meanwhile we already made improvements to accessibility and we have more on point issues (see the [accessibility tag](https://github.com/deltachat/deltachat-desktop/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility)). So I'll close this issue because there are no concrete actionable points.

@bmivzkrp you are invited to test again with the current version and make new issues about specific aspects where accessibility can be improved.

PS: we recently had an accessibility quick scan, they will soon give us an report of their findings.
